packages/compatibility/backend/sirius-components-compatibility/src/main/java/org/eclipse/sirius/components/compatibility/forms/ViewExtensionDescriptionConverter.java

```diff
@@ -72,13 +72,12 @@ public FormDescription convert(ViewExtensionDescription viewExtensionDescription
         PageDescriptionConverter pageDescriptionConverter = new PageDescriptionConverter(interpreter, this.identifierProvider, this.semanticCandidatesProviderFactory);
         GroupDescriptionConverter groupDescriptionConverter = new GroupDescriptionConverter(interpreter, this.objectService, this.identifierProvider, this.modelOperationHandlerSwitchProvider);
 
-        // @formatter:off
         Map<org.eclipse.sirius.properties.GroupDescription, GroupDescription> siriusGroup2SiriusWebGroup = new HashMap<>();
-        List<GroupDescription> groupDescriptions = viewExtensionDescription.getCategories().stream()
+
+        viewExtensionDescription.getCategories().stream()
```

Comment:
Now that `Groups` are only contained by `Pages`, may be the conversion of `Groups` should be done in `PageDescriptionConverter`. It would allow to delete the `siriusGroup2SiriusWebGroup` map. What do you think about that? Did I miss something? If not, please do the change.

Reply:
After discussion with PCD, this compatibility layer will soon be remover and the `siriusGroup2SiriusWebGroup` is used to set the `idProvider` in the `GroupDescriptionConverter`, so we won't spend time to refactor this part.